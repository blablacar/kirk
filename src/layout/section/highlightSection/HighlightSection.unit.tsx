import React from 'react'
import renderer from 'react-test-renderer'

import { HighlightSection } from './index'

describe('HighlightSection', () => {
  it('should render default highlight section', () => {
    const section = <HighlightSection>default highlight section</HighlightSection>
    const renderedSection = renderer.create(section).toJSON()
    expect(renderedSection).toMatchSnapshot()
  })
})
