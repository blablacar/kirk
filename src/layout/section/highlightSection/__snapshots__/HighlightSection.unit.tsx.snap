// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HighlightSection should render default highlight section 1`] = `
.c0 {
  padding: 24px;
  background-color: #054752;
  color: #FFF;
}

.c1 {
  margin-left: auto;
  margin-right: auto;
  max-width: 1016px;
}

<section
  className="c0"
>
  <div
    className="c1"
  >
    default highlight section
  </div>
</section>
`;
